website/pages/en/arbitrum/l2-transfer-tools-faq.mdx

```diff
@@ -38,55 +48,143 @@ To transfer your subgraph, you will need to complete the following steps:
 
 You can initiate your transfer from the [Subgraph Studio](https://thegraph.com/studio/), [Explorer,](https://thegraph.com/explorer) or any subgraph details page. Click the "Transfer Subgraph" button in the subgraph details page to start the transfer.
 
-## How long do I need to wait until my subgraph is transferred
+### How long do I need to wait until my subgraph is transferred
 
 The transfer time takes approximately 20 minutes. The Arbitrum bridge is working in the background to complete the bridge transfer automatically. In some cases, gas costs may spike and you will need to confirm the transaction again.
 
-## Will my subgraph still be discoverable after I transfer it to L2?
+### Will my subgraph still be discoverable after I transfer it to L2?
 
 Your subgraph will only be discoverable on the network it is published to. For example, if your subgraph is on Arbitrum One, then you can only find it in Explorer on Arbitrum One and will not be able to find it on Ethereum. Please ensure that you have Arbitrum One selected in the network switcher at the top of the page to ensure you are on the correct network.  After the transfer, the L1 subgraph will appear as deprecated.
 
-## Does my subgraph need to be published to transfer it?
+### Does my subgraph need to be published to transfer it?
 
 To take advantage of the subgraph transfer tool, your subgraph must be already published to Ethereum mainnet and must have some curation signal owned by the wallet that owns the subgraph. If your subgraph is not published, it is recommended you simply publish directly on Arbitrum One - the associated gas fees will be considerably lower. If you want to transfer a published subgraph but the owner account hasn't curated any signal on it, you can signal a small amount (e.g. 1 GRT) from that account; make sure to choose "auto-migrating" signal.
 
-## What happens to the Ethereum mainnet version of my subgraph after I transfer to Arbitrum?
+### What happens to the Ethereum mainnet version of my subgraph after I transfer to Arbitrum?
 
 After transferring your subgraph to Arbitrum, the Ethereum mainnet version will be deprecated. We recommend you update your query URL within 48 hours. However, there is a grace period in place that keeps your mainnet URL functioning so that any third-party dapp support can be updated.
 
-## After I transfer, do I also need to re-publish on Arbitrum?
+### After I transfer, do I also need to re-publish on Arbitrum?
 
 After the 20 minute transfer window, you will need to confirm the transfer with a transaction in the UI to finish the transfer, but the transfer tool will guide you through this. Your L1 endpoint will continue to be supported during the transfer window and a grace period after. It is encouraged that you update your endpoint when convenient for you.
 
-## Will there be a down-time to my endpoint while re-publishing?
+### Will there be a down-time to my endpoint while re-publishing?
```

Comment:
While we're at it, it might be worth editing this one to clarify that while there _should_ be no down time, it is possible that some brief downtime will occur depending on which Indexers were indexing on L1 and whether they keep Indexing it until the L2 subgraph is indexing. Just to set expectations that chances are low but it might happen.

Reply:
Added that in. Thank you!